File: php-slim/lib/Model/RequestTimeFilterDepartureSearch.php
<?php
/**
 * RequestTimeFilterDepartureSearch
 */
namespace OpenAPIServer\Model;

/**
 * RequestTimeFilterDepartureSearch
 */
class RequestTimeFilterDepartureSearch
{
    
    /** @var string $id */
    private $id;
    
    /** @var string $departureLocationId */
    private $departureLocationId;
    
    /** @var string[] $arrivalLocationIds */
    private $arrivalLocationIds;
    
    /** @var \OpenAPIServer\Model\RequestTransportation $transportation */
    private $transportation;
    
    /** @var int $travelTime */
    private $travelTime;
    
    /** @var \DateTime $departureTime */
    private $departureTime;
    
    /** @var \OpenAPIServer\Model\RequestTimeFilterProperty[] $properties */
    private $properties;
    
    /** @var \OpenAPIServer\Model\RequestRangeFull $range */
    private $range;
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/RequestTimeFilterDepartureSearch.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.RequestRangeFull
open traveltimeplatform.Model.RequestTimeFilterProperty
open traveltimeplatform.Model.RequestTransportation

module RequestTimeFilterDepartureSearch = 

  //#region RequestTimeFilterDepartureSearch


  type RequestTimeFilterDepartureSearch = {
    Id : string;
    DepartureLocationId : string;
    ArrivalLocationIds : string[];
    Transportation : RequestTransportation;
    TravelTime : int;
    DepartureTime : DateTime;
    Properties : RequestTimeFilterProperty[];
    Range : RequestRangeFull;
  }
  //#endregion
  